AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -71,10 +72,11 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
         }
 
         syncOperationQueue.isSuspended = false
+
+        dispatchSyncQueriesStarted(syncableModels)
```

Comment:
thinking about this some more and i think we should actually do the signal to say "sync queries started" and immediately release the suspend on the operation queue. This way, the sync query is not happening concurrent to the dispatch. 

```
dispatchSyncQueriesStarted(syncableModels)
syncOperationQueue.isSuspended = false
```

Reply:
Makes sense.